apps/website/src/routes/docs/headless/carousel/index.mdx

```diff
@@ -124,6 +124,13 @@ Qwik UI supports carousels without a scroller, which can be useful for condition
 
 Remove the `<Carousel.Scroller />` component to remove the scroller.
 
+## Vertical Direction
```

Comment:
Let's update the CSS scroll snapping section to include the updated CSS code for vertical mode.